onboarding-manager/pkg/cloudinit/99_infra.cfg

```diff
@@ -201,6 +201,19 @@ write_files:
       Subsystem sftp /usr/lib/openssh/sftp-server
       {{- end }}
    {{- end }}
+  {{- if .RUN_AS_STANDALONE }}
```

Comment:
@rranjan3 are we sure about this approach? IMO we should use custom cloud-init config of config-file